### DISCUSSION TITLE
Question: Dashboard Performance

### DISCUSSION THREAD

**Discussion by interpegasus** [railsadminteam/rails_admin#3601] [Q&A]
My app has a db with too many tables, and the dashboard page is a bit slow to load.
I would like to know if it is possible to improve performance by using cache on queries like record count?  Instead of removing all the stats. 

Thank you

**Reference**
https://github.com/sferik/rails_admin/wiki/Dashboard-action#disabling-record-count-bars

**Comment by RocKhalil** [comment]
@interpegasus 

I'd suggest you remove statistics and create your own dashboard.
File name: `app/views/rails_admin/main/dashboard.html.erb`
